03_011/src/utils/eventhandler.js

```diff
@@ -0,0 +1,66 @@
+import { actions } from '../redux/action';
+import contentApi from '../api';
+
+function checkBookmarkDiff(store, lists) {
+  if (window.location.hash === '#bookmark') return;
+  if (lists?.length) {
+    const {
+      state: { likeContents },
+    } = store.getState();
+
+    for (let list of lists) {
+      const idx = list.querySelector('div').id.replace('category-item-', '');
+      const { innerText: text } = list.querySelector('.bookmark-button');
+      const { length } = likeContents.filter((content) => content.idx === +idx);
+      if (text === '☆' && length) {
+        list.querySelector('.bookmark-button').innerText = '★';
+      } else if (text === '★' && !length) {
+        list.querySelector('.bookmark-button').innerText = '☆';
+      }
+    }
+  }
+}
+
+function checkClickBookmarkButton(className) {
+  return className === 'bookmark-button' ? true : false;
+}
+
+function onHandleClickBookmark(store, text, info) {
+  if (text === '☆') {
+    store.dispatch(actions.addBookmark(info));
+  } else {
+    store.dispatch(actions.removeBookmark(info));
+  }
```

Comment:
```suggestion
store.dispatch(actions.toggleBookmark(info));
```
이렇게 표현할 수 있도록 `toggleBookmark`를 만들어주면 어떨까요?